plugins/modules/panos_type_cmd.py

```diff
@@ -143,6 +143,7 @@
     from pandevice.errors import PanDeviceError
     import xmltodict
     import json
+    import xml
```

Comment:
Can you import just the error handler (ex. `from xml.parsers.expat import ExpatError`)?